src/components/directory-content.js

```diff
@@ -346,7 +346,10 @@ const DirectoryContent = () => {
     function typeCellRender(cellData) {
         const { rowData = {} } = cellData || {};
         const elementUuid = rowData['elementUuid'];
-        const objectType = rowData[cellData.dataKey];
+        let objectType = rowData[cellData.dataKey];
+        if (objectType === ElementType.PARAMETERS) {
+            objectType = cellData.rowData.description;
```

Comment:
It's weird to get the type from the description here. The subtype could be in the specificMetadata as it is done for the filters ?